lib/src/over_react_redux/over_react_flux.dart

```diff
@@ -0,0 +1,587 @@
+import 'dart:async';
+import 'dart:html';
+
+import 'package:collection/collection.dart';
+import 'package:meta/meta.dart';
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+import 'package:redux/redux.dart' as redux;
+import 'package:w_flux/w_flux.dart' as flux;
+
+import 'value_mutation_checker.dart';
+
+part 'over_react_flux.over_react.g.dart';
+
+@PropsMixin(keyNamespace: '')
+abstract class _$ConnectFluxPropsMixin<TActions> implements UiProps {
+  @override
+  Map get props;
+
+  TActions actions;
+}
+
+/// The actions that are associated with a given [InfluxStoreMixin] store
+/// during the construction of a [FluxToReduxAdapterStore].
+@visibleForTesting
+final Expando<dynamic> actionsForStore = Expando();
+
+/// The action used to keep Redux in sync with Flux when using a [FluxToReduxAdapterStore].
+class _FluxStoreUpdatedAction {
+  const _FluxStoreUpdatedAction();
+}
+
+/// A class that can be used to make a [flux.Store] compatible with [connectFlux]
+/// without adding the Redux implementation.
+///
+/// The usage is the same as [connectFlux] with the [FluxToReduxAdapterStore],
+/// but does not have the infrastructure for Redux `dispatch`ing.
+///
+/// __Note:__ Because this adapter store does not have a real Redux reducer,
+/// a `connected` component cannot trigger actions. If the library in the position to
+/// have Redux components, this class should not be used (implementing
+/// [FluxToReduxAdapterStore] instead) and the Redux boilerplate should be added.
+/// If needed, however, a `connect`ed component can receive updates from
+/// this implementation (just not trigger them).
+///
+/// __Example:__
+/// ```dart
+/// import 'package:w_flux/w_flux.dart' as flux;
+///
+/// class ExampleFluxStore extends flux.Store {
+///   FluxActions _actions
+///
+///   String _valueFromState = 'Default Value';
+///   String get valueFromState => _valueFromState;
+///
+///   ExampleFluxStore(this._actions) {
+///     triggerOnActionV2(_actions.fluxAction, _mutateValueFromState);
+///   }
+///
+///   void _mutateValueFromState(String newValue) {
+///     _valueFromState = newValue;
+///   }
+/// }
+///
+/// class FluxActions {
+///   final flux.Action<String> changeBackgroundColor = flux.Action();
+/// }
+///
+/// FluxActions actions = FluxActions();
+/// ExampleFluxStore store = ExampleFluxStore(actions);
+///
+/// ConnectFluxAdapterStore adapterStore = ConnectFluxAdapterStore<ExampleFluxStore, Null>(store, actions);
+/// ```
+///
+/// Related: [connectFlux], [FluxToReduxAdapterStore]
+class ConnectFluxAdapterStore<S extends flux.Store> extends redux.Store<S> {
```

Comment:
@greglittlefield-wf I changed the name from your suggestion (`FluxToReduxAdapterStoreWithoutInflux`) to this because it seemed like it's only usage would be to enable `connectFlux`, so it seemed like the name could reflect the focus a little more. If you envisioned this being more general purpose though, then let me know and I can change the name back to your suggestion!

Reply:
I like it!